src/AspNetCoreModuleV2/AspNetCore/OutOfProcessShimStaticHtml.htm

```diff
@@ -76,5 +76,7 @@ <h2> Troubleshooting steps: </h2>
         <h2> For more information visit:
             <a href="https://go.microsoft.com/fwlink/?LinkID=808681"> <cite> https://go.microsoft.com/fwlink/?LinkID=808681 </cite> </a>
         </h2>
+
+        %S
```

Comment:
What's this?

Reply:
Placeholder for `ANCM_ADDITIONAL_ERROR_PAGE_TEXT `